tests/TestCase/Http/Client/ResponseTest.php

```diff
@@ -244,6 +244,77 @@ public function testIsOk()
         $this->assertFalse($response->isOk());
     }
 
+    /**
+     * provider for isSuccess.
+     *
+     * @return array
+     */
+    public static function isSuccessProvider()
+    {
+        return [
+            [
+                true,
+                new Response([
+                    'HTTP/1.1 200 OK',
+                    'Content-Type: text/html'
+                ], 'ok')
+            ],
+            [
+                true,
+                new Response([
+                    'HTTP/1.1 201 Created',
+                    'Content-Type: text/html'
+                ], 'ok')
+            ],
+            [
+                true,
+                new Response([
+                    'HTTP/1.1 202 Accepted',
+                    'Content-Type: text/html'
+                ], 'ok')
+            ],
+            [
+                true,
+                new Response([
+                    'HTTP/1.1 203 Non-Authoritative Information',
+                    'Content-Type: text/html'
+                ], 'ok')
+            ],
+            [
+                true,
+                new Response([
+                    'HTTP/1.1 204 No Content',
+                    'Content-Type: text/html'
+                ], 'ok')
```

Comment:
Why does 204 have content?

Reply:
Sorry, just copy the test data from `testIsOk`. Shall I also remove the content from the response there?